### ISSUE TITLE
Uncaught TypeError: Cannot read property 'animating' of null

### ISSUE THREAD

**Issue by briandeweese** [jeremyramin/terminal-plus#86]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.2.0
**System**: Mac OS X 10.10.5
**Thrown From**: [terminal-plus](https://github.com/jeremyramin/terminal-plus) package, v0.14.0

### Stack Trace

Uncaught TypeError: Cannot read property 'animating' of null

```
At /Users/briandeweese/.atom/packages/terminal-plus/lib/status-bar.coffee:27

TypeError: Cannot read property 'animating' of null
    at atom-workspace.subscriptions.add.atom.commands.add.terminal-plus:next (/Users/briandeweese/.atom/packages/terminal-plus/lib/status-bar.coffee:27:34)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:533:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:356:22)
    at KeymapManager.module.exports.KeymapManager.terminatePendingState (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:507:14)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:383:23)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:99:36)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:3:61)
```

### Commands

```
     -1:41.4.0 grammar-selector:show (atom-text-editor.editor)
     -1:03.4.0 terminal-plus:next (atom-pane.pane.active)
     -0:11.6.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:03.0 window:focus-pane-on-right (atom-pane.pane.active)
     -0:00.5.0 terminal-plus:next (atom-pane.pane.active)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "turbo-javascript"
    ]
  },
  "terminal-plus": {
    "core": {},
    "style": {}
  }
}
```

### Installed Packages

```coffee
# User
color-picker, v2.0.13
language-gradle, v0.0.3
react, v0.12.10
react-snippets, v0.2.0
terminal-plus, v0.14.0

# Dev
No dev packages
```

**Comment by jeremyramin** [comment]
Thanks for reporting this, I will get right on it.

**Comment by jeremyramin** [comment]
Fixed as of v0.14.2
